packages/core/src/components/Pagination/Pagination.tsx

```diff
@@ -34,7 +37,12 @@ export const Pagination: FC<PaginationProps> & WithStyle = React.memo(
             else
                 links.push(
                     <PageNumberButton key={i} onClick={onClickHandler(linkItems[i])} isActive={linkItems[i] === currentPage}>
-                        <Text textAlign="center" textWeight={linkItems[i] === currentPage ? 'Strong' : 'Medium'}>
+                        <Text
+                            textAlign="center"
+                            textWeight={
```

Comment:
You can just pass `isSelected` prop to the `PageNumberButton` component and write the logic over there.

Reply:
checking `isActive` prop